.github/workflows/rust.yml

```diff
@@ -139,6 +139,7 @@ jobs:
       with:
         repository: proxy-wasm/proxy-wasm-rust-sdk
         path: proxy-wasm-rust-sdk
+        ref: v0.1.4
```

Comment:
The build for `v0.1.4` seems broken, so unless you want to investigate, I don't think it makes sense to pin to it here.

We can pin to `v0.2.1` or something in one of the later PRs.